Portfolio/index.css

```diff
@@ -1 +1,64 @@
-/* Write your code below for PORTFOLIO */
+/* Write your code below for PORTFOLIO */
+*{
+      color: #515050;
+          margin: 0%;
+    padding: 0%;
+}
+.cont1{
+    background-color: #90A4AE;
+    margin: 0%;
+    padding: 10px;
+  
+}
+nav ul{
+    padding: 0px;
+    margin: 0px;
+}
+nav li{
+    list-style-type: none;
+}
+.button{
+    border-radius: 20px;
+    padding: 10px;
+    border-color: #515050;
+}
+img{
+    border-radius: 50%;
+}
+form{
+    border: 2px solid #ffffff66;
+    background-color: #ffffff66;
+    display: inline-block;
+    padding: 5px 40px;
+    border-radius: 15px;
+}
+form h4{

```

Comment:
Consider adding a space between the property and value, like 'background-color: #ffffff66;' for consistency